docs/terminology.md

```diff
@@ -0,0 +1,72 @@
+# Terminology
+
+This document describes terms and concepts often used within the APM ecossystem
+
+
+#### APM
+Application Performance Monitoring is the concept of profiling and monitoring systems
+
+#### APM dashboards
+Custom Kibana dashboards made for APM. These used to be bundled with Kibana but are now located in the [apm-contrib repo](https://github.com/elastic/apm-contrib/tree/master/apm-ui)
+
+#### APM UI
+The curated UI in Kibana. This is part of the Elastic Basic License.
+[Further reading](https://www.elastic.co/guide/en/kibana/7.3/xpack-apm.html)
+
+#### Agent 
+An APM agent lives inside an application and will automatically collect APM data (transactions, spans, metrics and errors) and send it to the APM Server.
+[Further reading]](https://www.elastic.co/guide/en/apm/get-started/current/components.html#_apm_agents)
+
+#### APM Server
+The APM Server receives data from the Elastic APM agents and stores the data into Elasticsearch.
+[Further reading](https://www.elastic.co/guide/en/apm/get-started/current/components.html#_apm_server)
+
+#### Instrumentation
+To be able to monitor an application it needs to be _instrumented_. Instrumentation can happen automatically (the Elastic APM agents intrument many things automatically) or manually.
```

Comment:
```suggestion
To be able to monitor an application it needs to be _instrumented_. Instrumentation can happen automatically (the Elastic APM agents instrument many things automatically) or manually.
```

Definitions of automatic and manual are probably useful, perhaps:

automatic: Zero or minimal code changes needed for agent to instrument application
manual: Code changes required, helper libraries are often provided to simplify changes.

Do things like java auto-attach fit here as well?